setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import  setup
+from setuptools import find_packages
+
+REQUIREMENTS = [i.strip() for i in open("requirements.txt").readlines()]
+
+setup(
```

Comment:
Please, add also an entrypoint for the application so it can be easily executed after the installation.